5-0-functional-programming/src/test/java/com/bobocode/lambda_math_functions/FunctionsTest.java

```diff
@@ -0,0 +1,96 @@
+package com.bobocode.lambda_math_functions;
+
+import com.bobocode.InvalidFunctionNameException;
+import org.junit.jupiter.api.*;
+
+import java.util.function.Function;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+
+@TestMethodOrder( MethodOrderer.OrderAnnotation.class)
+public class FunctionsTest {
+    private com.bobocode.FunctionMap<Integer, Integer> integerFunctionMap;
+
+    @BeforeEach
+    public void init() {
+        integerFunctionMap = com.bobocode.Functions.intFunctionMap();
+    }
+
+    @Test
+    @Order(7)
+    void squareFunction() {
+        Function<Integer, Integer> squareFunction = integerFunctionMap.getFunction("square");
+
+        int actualResult = squareFunction.apply(5);
+
+        assertEquals(25, actualResult);
```

Comment:
You should replace all the assertions by "AssertJ" corresponding to [Migration Guide](https://bobocode.atlassian.net/wiki/spaces/UJE/pages/406290441/Migration+guide) .

Reply:
I corrected it, at least I think so. 
I do not fully understand what I am doing and how to do it correctly.
But I hope this time is what it was about it.